### PR TITLE
A couple trivial cleanups

### PR DESCRIPTION
**Pull request by euank**
Nothing too complicated here, a clarifying comment and nixing some unused lines

### CODE REVIEW

File: config/atom/atom.go [hunk 1]
```diff
@@ -52,9 +52,9 @@ func Open(name string) (File, error) {
 }
 
 type file struct {
-	to    string
-	tmp   *os.File
-	dirty bool
+	to       string
+	tmp      *os.File
+	writeErr bool
```
Comment (gsamokovarov) [inline config/atom/atom.go]: I actually like `dirty`  as it hints it's a boolean and that the state is not right. When I read `writeErr`, I expect an error on first sight, but the value turns out to be a boolean.

File: config/entries.go [hunk 2]
```diff
@@ -16,10 +16,7 @@ func (c *fileConfig) ReadEntries() (entries scoring.Entries, err error) {
 	}
 	defer file.Close()
 
-	if err = jsonio.Decode(file, &entries); err != nil {
-		return
-	}
-
+	err = jsonio.Decode(file, &entries)
```
Comment (gsamokovarov) [inline config/entries.go]: I'm fine with this change, though. Can we just leave this one?

File: config/atom/atom.go (hunk 1 above)
Comment (euank) [reply config/atom/atom.go]: Agreed `writeErr` isn't quite right either.

Do you think any of the following  seem better than dirty to you?

* `errored`
* `writeErrored`
* `writeErrOccured`
* `writeFailed`
* `noCommit`
* `dontRename`
* `badWrite`
* `invalidState`

If not, I'll be happy to leave it as is or add a clarifying comment over it in the struct.

Comment (gsamokovarov) [reply config/atom/atom.go]: I like `badWrite`, but to be honest, I don't think `dirty` is so confusing here. Maybe you can put the comment over it.

**Comment by euank** [comment]
Updated to just add a clarifying comment

**Comment by gsamokovarov** [comment]
Thanks, @euank!

**Pull request merged**